### PR TITLE
[UI] Ability to maximize the Application and Pattern Editor

### LINKED ISSUE #3914: [UI] Ability to maximize the Application and Pattern Editor
#### Current Behavior
Long pattern and application files are easier to edit when the user can maximize (and restore back to original modal size) the editor modal.

#### Implementation
1. choose one of the two matching sets of icons below (https://react-icons.github.io/react-icons/search?q=maxim)
1. place maximize/restore icon in upper right hand corner of modal

#### Acceptance Tests
1. When modal is orginal size show maximize button.
1. When modal is maximized show restore button.

#### Mockups
Screen Shot 2021-08-17 at 1 26 49 PM
Screen Shot 2021-08-17 at 1 34 16 PM
Screen Shot 2021-08-17 at 1 34 08 PM

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and Resources

- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🖥 [Contributing to Meshery UI](https://docs.meshery.io/project/contributing-ui)
- 🙋🏾🙋🏼 Questions: [Layer5 Discussion Forum](https://discuss.layer5.io) and [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by Abhishek-kumar09**
Signed-off-by: Abhishek-kumar09 <abhimait1909@gmail.com>

**Description**
Add fullScreen buttons to maximise/minimise dialog

This PR fixes #3914 

Screnshots: 
Screenshot from 2021-08-29 20-58-50
Screenshot from 2021-08-29 20-58-55

**Notes for Reviewers**

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: ui/components/MesheryPatterns.js [hunk 1]
```diff
@@ -182,7 +227,8 @@ function MesheryPatterns({
 
   const searchTimeout = useRef(null);
   /**
-   * fetch patterns when the page loads
+   * fetch patterns when the page loadsOh that's okay @warunicorn19
```
Comment (warunicorn19) [inline ui/components/MesheryPatterns.js:230]: seemed to have commented this by mistake😅

Comment (Abhishek-kumar09) [reply ui/components/MesheryPatterns.js:230]: How I got these changes? I am confused. Hell, When did I do this

Comment (Abhishek-kumar09) [reply ui/components/MesheryPatterns.js]: Oh, I think it's a surprise from ctrl+c and ctr+v

Comment (Abhishek-kumar09) [reply ui/components/MesheryPatterns.js]: Removed

**Comment by warunicorn19** [comment]
@Abhishek-kumar09 you've only added it to the patterns right? not applications yet?

**Comment by Abhishek-kumar09** [comment]
> @Abhishek-kumar09 you've only added it to the patterns right? not applications yet?

No I haven't added in application. Actually Applications and filters needs to catch up all the UI change that patterns have. Maybe we can cover it in another PR? or this?

**Comment by warunicorn19** [comment]
Yup let's do that in a different PR then.

**Review by leecalcote — APPROVED** [review]

**Pull request merged**